bin/install.sh

```diff
@@ -136,7 +136,7 @@ command="$1"
 profile=$2
 shift;
 shift;
-if [ "$3" == "logout" ]
+if [ "$1" == "logout" ]
```

Comment:
Can you explain how you are calling `withokta`. I don't understand why we even need to check why `$1` or `$3` (though I do agree `$3` in this case is incorrect and will reference an empty string).  

If you just call:
`withokta logout profile` command will already be logout

Reply:
I'm using the [awscli command ](https://github.com/oktadeveloper/okta-aws-cli-assume-role/blob/master/bin/install.sh#L167). When I execute `awscli logout` 'awscli' wrapper script run `withokta aws default logout` and in this moment the withokta script doesn't work as expected.

Reply:
I updated the PR name and description. Now, they are more accurate.